packages/cfd/src/Constants/cfd-shared-strings.js

```diff
@@ -24,7 +24,7 @@ export const general_messages = {
         } else if (platform === CFD_PLATFORMS.MT5) {
             if (is_eu) {
                 return localize(
-                    'Trade CFDs on forex, stocks, stock indices, synthetic indices, and commodities with leverage.'
+                    'Trade CFDs on forex, stocks, stock indices, derived, cryptocurrencies, and commodities.'
```

Comment:
isn't `commodities` part of `Derived` after restructuring markets?

Reply:
Nope, only baskets and synthetics